src/main/java/com/jalasoft/jfc/model/VideoConverter.java

```diff
@@ -49,9 +49,6 @@ public boolean convert(VideoParam videoParam) throws IOException, InterruptedExc
             /**
              * This statement adds fFmpeg value to command string. */
```

Comment:
Delete all code documentation inside a method

Reply:
ok, i will do it.

Reply:
Tank you Juan

Reply:
it is done, thank you

Reply:
> Tank you Juan

you are welcome